packages/@glimmer/component/src/tracked.ts

```diff
@@ -283,19 +269,8 @@ export default class Meta {
   }
 }
 
-function combinatorForComputedProperties(meta: Meta, key: Key, dependencies: Key[] | void): Tag {
-  // Start off with the tag for the CP's own dirty state.
-  let tags: Tag[] = [meta.updatableTagFor(key)];
-
-  // Next, add in all of the tags for its dependencies.
-  if (dependencies && dependencies.length) {
-    for (let i = 0; i < dependencies.length; i++) {
-      tags.push(meta.tagFor(dependencies[i]));
-    }
-  }
-
-  // Return a combinator across the CP's tags and its dependencies' tags.
-  return combine(tags);
+function combinatorForComputedProperties(meta: Meta, key: Key): Tag {
+  return combine([meta.updatableTagFor(key)]);
```

Comment:
I'm not sure we even need to call `combine` here. If there is only 1 item in the array there isn't anything to combine.